### DISCUSSION TITLE
About elastic moduli along this direction of each grain

### DISCUSSION THREAD

**Discussion by zz123-fix** [mtex-toolbox/mtex#2232] [Ask Anything]
![ee](https://github.com/user-attachments/assets/c8cfb32d-ceb7-44cd-be2e-41e7aa798116)
Which function in mtex can provide the elastic moduli along this direction of each grain mentioned in this article?

**Comment by Tijmenvermeij** [comment]
A good starting point can be found here: https://mtex-toolbox.github.io/AnisotropicTheory.html
You can rotate the stiffness tensor using a crystal/grain orientation and then calculate the modulus in a certain direction.

Tijmen
